doc/attr.xml

```diff
@@ -902,6 +902,36 @@ gap> DigraphShortestDistances(D);
 </ManSection>
 <#/GAPDoc>
 
+<#GAPDoc Label="UnweightedBellmanFord">
+<ManSection>
+  <Attr Name="UnweightedBellmanFord" Arg="digraph","source"/>
+  <Returns>A list of integers or <K>fail</K>.</Returns>
+  <Description>
+    If <A>digraph</A> is a digraph with <M>n</M> vertices, then this
+    function returns a list with two sublists of <M>n</M> entries, where each entry is
+    either a non-negative integer, or <K>fail</K>.  <P/>
+
+    If there is a directed path from <A>source</A> to vertex <C>i</C>, then
+    the value of the entry i in the first sublist is the length of the shortest such directed
+    path and the entry i in the second sublist is the vertex preceding the vertex of that entry. If no such directed path exists,     then the value of i is <C>fail</C>.  We use the convention that the distance from every vertex to
```

Comment:
I'm not sure what `the vertex preceding the vertex of that entry` means, can you please clarify? And also please wrap the line so that it is at most 80 chars long.